### PR TITLE
Adding change log for 10.4.0

### PR DESCRIPTION
**Pull request by mjhenkes**

### CODE REVIEW

File: content/_changelogs/10.4.0.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  supported. Addresses
+  [#21476](https://github.com/cypress-io/cypress/issues/21476),
+  [#21307](https://github.com/cypress-io/cypress/issues/21307),
+  [#22953](https://github.com/cypress-io/cypress/issues/22953), &
+  [#21474](https://github.com/cypress-io/cypress/issues/21474).
+- Cypress will now display a banner with a corrective action when your project
+  uses a projectId that cannot be found in the Cypress Dashboard and when you do
+  not have access to the current project in the Cypress Dashboard. Addresses
+  [#22637](https://github.com/cypress-io/cypress/issues/22637).
+- Cypress will now
```
Comment (tbiethman) [inline content/_changelogs/10.4.0.md:28]: Can remove this

Comment (mjhenkes) [reply content/_changelogs/10.4.0.md:28]:
```suggestion
```

**Comment by emilyrohrbough** [comment]
Not sure if it's not worthy but these were tagged with 10.3.2/I missed a tag:
- https://github.com/cypress-io/cypress/issues/21394
- https://github.com/cypress-io/cypress/issues/21399

- Visual enhancements were made the `cy.session()` logs that are shown in the Command Log. Addressed [#21394](https://github.com/cypress-io/cypress/issues/21394) and [#21399](https://github.com/cypress-io/cypress/issues/21399).
- Visual enhancements were made to the alias and intercept tags shown in the Command log. Addressed in [#22857](https://github.com/cypress-io/cypress/pull/22857).

**Comment by mjhenkes** [comment]
@emilyrohrbough, i skipped the style and verbiage updates. They didn't seem important enough to me, but if you disagree i can add them in.

**Comment by emilyrohrbough** [comment]
Nope, totally fine with skipping! Just wanted to make sure they weren't missed.

**Comment by tbiethman** [comment]
I don't see the Electron 19.0.8 update: https://github.com/cypress-io/cypress/pull/22775

It didn't have an associated issue, but it's marked for 10.4.0

**Comment by mjhenkes** [comment]
@tbiethman I missed that one. I'll get it added.

**Review by tbiethman — APPROVED** [review]

File: content/_changelogs/10.4.0.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+## 10.4.0
+
+_Released 8/1/2022_
```
Comment (marktnoonan) [inline content/_changelogs/10.4.0.md:3]:
```suggestion
_Released 8/2/2022_
```

Comment (mjhenkes) [reply content/_changelogs/10.4.0.md]: 😅 , oops

File: content/_changelogs/10.4.0.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+- Fixes issue where cookies were not handled within cy.origin for requests other
+  than the AUT page request. Fixes
+  [#22952](https://github.com/cypress-io/cypress/issues/22952) &
+  [#21474](https://github.com/cypress-io/cypress/issues/21474).
+- Tooltips are now correctly displayed behind modals. Fixes
+  [#22595](https://github.com/cypress-io/cypress/issues/22595).
+- Fixed a hang when sending video to the dashboard from the electron browser.
+  Fixes [#17627](https://github.com/cypress-io/cypress/issues/17627).
+- Cypress will now throw an error when we receive a unexpected return value from
```
Comment (marktnoonan) [inline content/_changelogs/10.4.0.md:39]:
```suggestion
- Cypress will now throw an error when we receive an unexpected return value from
```

File: content/_changelogs/10.4.0.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+  a custom config file. Fixes
+  [#21948](https://github.com/cypress-io/cypress/issues/21948).
+- Cypress will now warn when detecting unsupported dependencies for component
+  testing. Fixes [#22025](https://github.com/cypress-io/cypress/issues/22025).
+- cy.type() can now be used to enter datetime values with seconds, to an input
+  of type datetime-local, for example 1959-09-13T10:12:13. Fixes
+  [#22884](https://github.com/cypress-io/cypress/issues/22884).
+- Next.js in Cypress is now allowed to import global styles. Fixes
+  [#22525](https://github.com/cypress-io/cypress/issues/22525)
+- Fixed an issue where 'additionalIgnorePatter' was not allowed to be an array
```
Comment (marktnoonan) [inline content/_changelogs/10.4.0.md:49]:
```suggestion
- Fixed an issue where 'additionalIgnorePattern' was not allowed to be an array
```

**Review by marktnoonan — APPROVED** [review]

**Pull request merged**